libs/wingsdk/src/target-tf-aws/api.ongetrequest.inflight.ts

```diff
@@ -1,14 +1,27 @@
-import { IApiOnGetHandlerHandlerClient } from "./api";
+// TODO: remove this line and discuss with @Chriscbr about how to handle this
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
```

Comment:
I'm OK leaving the eslint warning if it compiles :) We could also just hand-copy a simple version of the API / the fields we care about and duplicate it into our source code (I feel like it's unlikely they introduce breaking changes since it's a live AWS service)

Reply:
I see two option how to resolve this error. Either move the dependecy from devDependency to dependency or add a exclusion for @types packages in the eslint rule

Reply:
I like the second option (first is also ok).

Another thing you could try is:
```ts
import type { ... } from "aws-lambda";
```

Reply:
the import type version works